starboard/shared/posix/time_zone_get_name.cc

```diff
@@ -12,13 +12,59 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "starboard/common/log.h"
 #include "starboard/time_zone.h"
 
-#include <time.h>
+#define TZDEFAULT "/etc/localtime"
```

Comment:
Is there a guarantee that this file will always exist? I guess the system clock depends on it so the answer is most likely yes?

Reply:
I think under Linux this should always be true. Maybe I should put this under a separate path, after the AppleTV fix is ready(so that tvOS does not depend on posix time_zone_get_name.cc)